plugins/backend-editor/src/Cardano/BM/Backend/Editor.lhs

```diff
@@ -19,7 +19,7 @@ module Cardano.BM.Backend.Editor
 import           Prelude hiding (lookup)
 import qualified Control.Concurrent.Async as Async
 import           Control.Concurrent.MVar (MVar, newEmptyMVar, putMVar, swapMVar, readMVar, withMVar)
-import           Control.Exception.Safe (SomeException, catch)
+import           Control.Exception.Safe (IOException, SomeException, catch)
```

Comment:
is "SomeException" still used, or can it be left out from import?
(also in other source files)

Reply:
In some places, it is, in places where the exceptions are rethrown or have no critical consequences.